.github/workflows/main.yml

```diff
@@ -15,18 +15,40 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - name: Setup Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/cache@v2
+        with:
+          path: ~/swig
+          key: ${{ runner.os }}-swig-pr1746
       - name: Setup SWIG
         run: |
-          git clone -b pr/new-node-fixes https://github.com/yegorich/swig;
-          cd swig;
-          ./autogen.sh;
-          ./configure --prefix=$HOME/swig;
-          make;
-          make install;
-      - name: Run.me
+          if [ ! -x ~/swig/bin/swig ]; then
+            ( git clone -b pr/new-node-fixes https://github.com/yegorich/swig;
+              cd swig;
+              ./autogen.sh;
+              ./configure --prefix=$HOME/swig;
+              make;
+              make install;
+            )
+          fi
+      # - name: Run.me
+      #   run: |
+      #     env PATH=~/swig/bin:$PATH SWIG_LIB=~/swig/share/swig/4.0.2 \
+      #       ./blst/bindings/node.js/run.me
+
+      - name: Copy patch files
         run: |
-          env PATH=~/swig/bin:$PATH SWIG_LIB=~/swig/share/swig/4.0.2 \
-            ./blst/bindings/node.js/run.me
+          cp blst_wrap.py blst/bindings/node.js/blst_wrap.py
+          cp blst.hpp blst/bindings/blst.hpp
```

Comment:
Head of blst already has updated blst.hpp.

Reply:
And if you want to run on Windows, you're likely to be better off if you specify `shell: bash`.